src/components/BooksContainer/BookList/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../../styles/book.css';
+
+const temp = [];
+
+const Book = () => (
+  temp.map((el) => (
+    <div key={el.id} className="book-container d-flex">
+      <div className="general-info d-flex">
+        <span className="genre">{el.genre}</span>
+        <h3 className="title">{el.title}</h3>
+        <span className="author">{el.author}</span>
+        <ul className="book-management d-flex">
+          <li>Comments</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </div>
+      <div className="completion-detail">
+        <span className="completion">{el.completionPercentage}</span>
+      </div>
+      <div className="current-chapter d-flex">
+        <span className="current-chapter-span">Current Chapter</span>
+        <span className="current-chapter-value">{el.currentChapter}</span>
+        <button type="button">Update Changes</button>
+      </div>
+    </div>
```

Comment:
- [ ] The Book component should take book data as a prop and renders the title and the author. These should be passed from the BookList to the Book component

Reply:
@mke2111 Excuse me please but i'm planning to have that not as a prop but i'm planning to to recieve that array from the redux state
(reduxState = { arrOfbooks : [], })
 and since the whole project will be wrapped inside <Provider> at later stage shouldn't I be able to access it from Book.js component directly and modify it using action and reducers which I will prepare for the matter ?

Reply:
That is another option you may consider but since this is one of the project requirements, you have to pass props to the Book component.

Reply:
so if I understood correctly what I need to do is get the temp array from book.js and place it in BookList and then pass it back to the Book component as Prop right ?